pkg/api/accessrule.go

```diff
@@ -127,7 +127,7 @@ func (a *API) AdminGetAccessRule(w http.ResponseWriter, r *http.Request, ruleId
 // (POST /api/v1/access-rules/{ruleId})
 func (a *API) AdminUpdateAccessRule(w http.ResponseWriter, r *http.Request, ruleId string) {
 	ctx := r.Context()
-	var updateRequest types.UpdateAccessRuleRequest
+	var updateRequest types.CreateAccessRuleRequest
```

Comment:
Is there any reason we are changing `UpdateAccessRuleRequest` to `CreateAccessRuleRequest` . Seems like `UpdateAccessRuleRequest` variable naming was more clear given that we are actually trying to update access rule.

Reply:
Because these types are identical I find it makes the typing simpler for the frontend and backend.
If these types diverge again we could switch back to a different type

Reply:
Maybe we could make it more generic "AccessRuleRequest"